node/alphanet/src/lib.rs

```diff
@@ -235,16 +221,15 @@ mod tests {
             None,
             3034,
             vec![bob.node_info.clone()],
-            chain_spec,
+            chain_spec.clone(),
         );
 
+        let (mut beacon_block, mut shard_block, shard_extra) =
+            alice.client.prepare_block(1, ChainPayload::default());
+        beacon_block.signature.authority_mask = vec![true; chain_spec.initial_authorities.len()];
+        shard_block.signature.authority_mask = vec![true; chain_spec.initial_authorities.len()];
         let (mut beacon_block, mut shard_block) =
-            match alice.client.try_produce_block(1, ChainPayload::default()) {
-                BlockProductionResult::Success(beacon_block, shard_block) => {
-                    (beacon_block, shard_block)
-                }
-                _ => panic!("Should produce block"),
-            };
+            alice.client.try_import_produced(beacon_block, shard_block, shard_extra);
         // Sign by bob to make this blocks valid.
         beacon_block.add_signature(&beacon_block.sign(bob.signer()), 1);
```

Comment:
why is block signed after importing? let's but that before `try_import_produced`?